modules/dcache/src/main/java/org/dcache/pool/statistics/IoStatistics.java

```diff
@@ -0,0 +1,205 @@
+package org.dcache.pool.statistics;
+
+import java.util.ArrayList;
+import java.util.Collections;
+
+/**
+ * This class stores statistics about read an write processes
+ */
+
+public class IoStatistics {
+
+    private long _readRequestNum = 0;
+    private long _readBytes;
+    private ArrayList<Double> _readSpeeds = new ArrayList<>();
+    private double _minReadSpeed;
+    private double _maxReadSpeed;
+    private double _avgReadSpeed;
+    private double _95ReadSpeed;
+    private long _totalReadTime;
+
+    private long _writeRequestNum = 0;
+    private long _writtenBytes;
+    private ArrayList<Double> _writeSpeeds = new ArrayList<>();
```

Comment:
I agree, maybe this should be substituted for 
http://commons.apache.org/proper/commons-math/javadocs/api-3.5/org/apache/commons/math3/stat/descriptive/rank/Percentile.html